docs/pages/gas-manager-admin-api/conditional-gas-sponsorship.mdx

```diff
@@ -0,0 +1,72 @@
+---
+title: Conditional gas sponsorship
+description: Step-by-step guide to sponsor gas for select transactions and users.
+url: https://docs.alchemy.com/reference/conditional-gas-sponsorship
+slug: reference/conditional-gas-sponsorship
+---
+
+In this guide, you’ll learn how to enable **conditional gas sponsorship** to sponsor gas only for select transactions or users. Ex:
+
+- Sponsor gas only for swaps or txs that interact with your contracts.
+- Sponsor gas only for high-ROI users such as power users of your app or users with proof of humanity.
+
+<Info>
+  **\[Recommended]** Use our [SDK](https://www.alchemy.com/docs/wallets) to
+  create and use wallets. The SDK handles all complexity for you, making
+  development faster and easier.
+</Info>
+
+If you want to use APIs directly, follow these steps.
```

Comment:
This doesn't really apply here as our SDK doesn't help with webhooks at all. We could either remove this or put it as part of step 3.